modular_skyrat/code/datums/mutations/body.dm

```diff
@@ -0,0 +1,65 @@
+/datum/mutation/human/thickskin
+	name = "Thick skin"
+	desc = "The user's skin acquires a leathery texture, and is more resilient to harm."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>Your skin feels dry and heavy.</span>"
+	text_lose_indication = "<span class='notice'>Your skin feels soft again...</span>"
+	difficulty = 18
+	instability = 25
+
+/datum/mutation/human/strong/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	if(owner.dna.species)
+		owner.dna.species.brutemod *= 0.75
+		owner.dna.species.burnmod *= 0.9
+
+/datum/mutation/human/strong/on_losing(mob/living/carbon/human/owner)
+	if(..())
+		return
+	if(owner.dna.species)
+		owner.dna.species.brutemod = initial(owner.dna.species.brutemod)
```

Comment:
Dont do it this way it nulls a few other things

Reply:
Corrected.